app/uk/gov/hmrc/hmrcemailrenderer/templates/mods/mods_import_declaration.scala.html

```diff
@@ -21,9 +21,10 @@
   goods.map { i =>
     val category = params(s"goodsCategory_$i")
     val quantity = params(s"goodsQuantity_$i")
-    val country = params(s"goodsCountry_$i")
+    val country = params.get(s"goodsCountry_$i")
```

Comment:
i think this `country` line breaks - also we dont need this for imports

Reply:
we need 'country' to allow it to still work with the current production version of our code. If DC release with without 'country' supported and we don't upgrade at the same time then out system will stop.

Reply:
makes sense, but let's say we have both `country` and `goodsProducedInEu`  in the prod templates then one of these  `params.get` calls would fail irrespective of what version of our service we have in production  because in any version we pass only one of these from our BE, no ?  or did i miss something ?